### ISSUE TITLE
Badge suggestions

### ISSUE THREAD

**Issue by acadet** [rails-school/school#129]
Here are some badge suggestions. Would be fun to add a description too. Also, if Edi or another designer can create some icons would be amazing.

Codewars challenges:

* Bleu-bite - 1 - `Sir, yes sir!`
* Footman - 5 - `Hey, you are not a virgin anymore`
* Ninja - 15 - `Whoah, you can vanish. Stealth mode. Magic powder.`
* Gladiator - 35 - `Code fears you. Too many cocky challenges passed away over your keyboard.`
* Mangler - 50 - `Crowd praises you.`
* Dovahkiin - 100 - `FUS RO DAH!`

If it's possible to spot Codewars rank, we can create suited badges.

Number of attendances:
* Jar Jar Binks - 1 - `Nobody knows why are you around, but you are.`
* Ewok - 5 - `You are tiny and cute. People love you at first sight. However, you still fight with rocks and wood sticks.`
* Greedo - 15 - `Hey, you are a regular dude. You are close to inspire respect.`
* Boba Fett - 35 - `You rock! Even if you are not so agile, sometimes. Carry on!`
* Jabba - 50 - `You are a must-see, little hoodlums want to be your sidekick.`
* Han Solo - 100 - `Seriously? You really need to introduce that guy?`

Number of classes (for teachers):
* Ki Adi Mundi - 1 - `Welcome to the club!`
* Plo Koon - 5 - `Everybody has seen you once, knows your face but doesn't remember your name.`
* Qui Gon Jinn - 15 - `Damn. You a real chap. Even if you act weird from time to time`
* Obi Wan Kenobi - 35 - `May the force with you padawans`
* Mace Windu - 50 - `I am Samuel L Jackson motherfucker.`
* Yoda - 100 - `Nobody understands you, but you do not give a fuck. You are Yoda. Deal with that.`

**Comment by gkop** [comment]
I'd also like to have one for RailsBridge people that queries BridgeTroll (https://github.com/railsbridge/bridge_troll)

**Comment by cdale77** [comment]
Number of attendances badges are implemented. 
BridgeTroll merged my PR that provides an API to learn how many RailsBridge classes people attend. 
https://github.com/railsbridge/bridge_troll/pull/311

**Comment by ondine** [comment]
Aloha! I believe the badge(rs)
![il_570xn 256953605](https://cloud.githubusercontent.com/assets/3281541/6718494/56bf1f3c-cd71-11e4-8b92-0c4ad8acdb97.jpg) have a few bugs. 

Just received an Ewok badge in email via sendgrid for attending 5 classes, but when logged in it says: This user has attended 15 lessons. Theoretically should have Jar Jar, Ewok and Greedo.

Badge suggestions from GoT:
• Daenerys Targaryen
• Arya Stark
• Brienne of Tarth
• Sansa Stark
• Ygritte

**Comment by gkop** [comment]
Yup, this is a bug affecting other users, too. @cdale77 any idea what's going wrong?

**Comment by gkop** [comment]
@ondine looks like simply a glitch due to us having forgotten to run a migration on a new feature! Refresh your profile page.

**Comment by ondine** [comment]
Sweet, Glitch-B-Gone!
